### PR TITLE
Fix:Text collapses in Marketplace page 

### LINKED ISSUE #16471: [BUG]: Text collapses in Marketplace page
### Description:
In rocketchat-server application, during admin>marketplace page, the Details/Price list heading text boxes collapse.

### Steps to reproduce:
It appears anytime while opening the page:
Home>Hamburger menu>Options>Administration>Marketplace

### Expected behavior:
It has to be clear to read.

### Actual behavior:
Text boxes have collapsed and texts overlap.
image

### Server Setup Information:

- Version of Rocket.Chat Server: 2.4.2
- Operating System: Ubuntu 18.04.3 LTS 64bit
- Deployment Method: snap
- DB Replicaset Oplog: Enabled
- NodeJS Version: 8.17.0 x-64
- MongoDB Version: 3.6.14

### PR DESCRIPTION
**Pull request by ishriom53tyagi**
FIX

Closes #16471 

fixed Details and Price tag are overlapped in market place page when device screen is in tablet mode.

### CODE REVIEW

File: app/theme/client/imports/components/table.css [hunk 1]
```diff
@@ -215,6 +215,8 @@
 	font-size: 0.75rem;
 	font-weight: 500;
 	line-height: 1rem;
+	background-color: white;
+	padding-right: 10px;
```
Comment (sampaiodiego) [inline app/theme/client/imports/components/table.css]: Codacy Issue found: [Expected an empty line before property "padding-right" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5001518)

File: app/theme/client/imports/components/table.css [hunk 2]
```diff
@@ -215,6 +215,8 @@
 	font-size: 0.75rem;
 	font-weight: 500;
 	line-height: 1rem;
+	background-color: white;
```
Comment (sampaiodiego) [inline app/theme/client/imports/components/table.css]: Codacy Issue found: [Expected an empty line before property "background-color" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5001518)

Comment (ishriom53tyagi) [reply app/theme/client/imports/components/table.css]: yeah, I fixed this. hope it works fine now

File: app/theme/client/imports/components/table.css [hunk 3]
```diff
@@ -211,11 +211,14 @@
 	text-align: left;
 
 	color: var(--rc-color-primary-light);
+
```
Comment (sampaiodiego) [inline app/theme/client/imports/components/table.css]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5001518)

File: app/theme/client/imports/components/table.css [hunk 4]
```diff
@@ -211,11 +211,14 @@
 	text-align: left;
 
 	color: var(--rc-color-primary-light);
+	
+	padding-right: 10px;
```
Comment (sampaiodiego) [inline app/theme/client/imports/components/table.css]: Codacy Issue found: [Expected an empty line before property "padding-right" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5001518)

**Comment by gabriellsh** [comment]
I think you may have mixed a couple of commits up there. Is this right?

**Pull request closed without merging**